src/App.js

```diff
@@ -1,9 +1,16 @@
 import React from 'react';
-import './App.css';
+import NavTop from "./components/Pages/Nav-Top/Nav-Top";
+import MainProducts from "./components/Pages/Main-Products/Main-Products";
+import FooterTiles from "./components/Pages/Footer-Info/FooterTiles";
 
 function App() {
   return (
+    <>
       <h1>Handbags & Purses</h1>
+      <NavTop/>
+      <MainProducts/>
+       <FooterTiles/>
```

Comment:
Hoi, regel 12: mischien kan je app.js formatteren. Volgens aandachtspuntenlijst-react-handbags. 
CMD + option + L (Mac) 
of CTRL + alt + L (Windows) drukt?